integration/helpers/base_test.py

```diff
@@ -45,6 +45,13 @@
 
 STACK_NAME_PREFIX = "sam-integ-stack-"
 S3_BUCKET_PREFIX = "sam-integ-bucket-"
+FEATURE_TOGGLE_JSON_FILES = [
```

Comment:
Where is the file name of the non-FT json files stored?

Reply:
The file names for non-FT json files are not stored anywhere. The idea is if the test is related to feature toggle then it would use non-FT + feature_toggle.json file